src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,227 @@
-""" This is a dummy example """
+import requests
+import pandas as pd
+import time
+import os
+import matplotlib.pyplot as plt
 
-def main():
-    raise NotImplementedError
 
-if __name__ == "__main__":
-    main()
+# def main():
+#     raise NotImplementedError
+
+# if __name__ == "__main__":
```

Comment:
you should include this in anyscript that is intended to be run standalone: https://stackoverflow.com/questions/419163/what-does-if-name-main-do
so uncomment it. It's normally added at the end of the script